modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/routes/BoxesRoutes.scala

```diff
@@ -112,7 +119,14 @@ final class BoxesRoutes[
 
   private def getOutputsByAddressR: HttpRoutes[F] =
     interpreter.toRoutes(defs.getOutputsByAddressDef) { case (address, paging) =>
-      service.getOutputsByAddress(address, paging).adaptThrowable.value
+      (for {
+        spentBoxIds    <- mempool.getUSpentOutputsByAddress(address).adaptThrowable
+        mempoolOutputs <- mempool.getUOutputsByAddress(address).adaptThrowable
+        filteredOutputs <-
+          service
+            .getFilteredOutputsByAddress(address, paging, NonEmptyList.ofInitLast(spentBoxIds, BoxId("")))
```

Comment:
What is this `BoxId("")`?

Reply:
updated this to use Option[NonEmptyList], from .toNel